.github/workflows/old-compilers.yml

```
@@ -8,6 +8,7 @@ env:
   DEFAULT_SANITIZE_THREAD: OFF
   DEFAULT_SANITIZE_UB: OFF
   DEFAULT_SPINLOCK_LOOP: PAUSE
+  DEFAULT_STATS: ON
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)